packs/ansible/actions/command.yaml

```diff
@@ -0,0 +1,108 @@
+---
+name: "command"
+description: "Run ad-hoc ansible command (module)"
+runner_type: "local-shell-script"
+entry_point: "ansible.py"
+enabled: true
+parameters:
+  kwarg_op:
+    description: "Lock operator type to '--'"
+    type: string
+    default: "--"
+    immutable: true
+  sudo:
+    description: "Lock sudo, the behavior is controlled by ansible 'become_' options"
+    type: boolean
+    default: true
+    immutable: true
+  cwd:
+    description: "Working directory where the command will be executed in"
+    type: string
+  timeout:
+    description: "Action timeout in seconds. Action will get killed if it doesn't finish in timeout seconds"
+    type: integer
+    default: 600
+  hosts:
+    description: "Hosts"
+    type: string
+    position: 0
+  args:
```

Comment:
I believe `args` argument should be required for `command` and `command_local` action, since running those actions without any args doesn't make sense, right? Or am I missing something?